### ISSUE TITLE
Send email report on pipeline status 

### ISSUE THREAD

**Issue by FriederikeHanssen** [nf-core/awsmegatests#13]
It would be nice to have an additional email report send out to the developers in addition to nextflow tower. 
For each pipeline a central email could be created such `pipeline@nf-co.re`, which developers can subscribe to to get updates on the aws pipeline execution status. This can be managed using organisation wide Github secrets.

**Comment by ewels** [comment]
Whilst this is a nice idea, it would require me to manually maintain all of the pipeline-specific email lists for now. This isn't something that's easy to automate sadly, so I'm not super keen.

After further discussion on Slack, I think the easiest short-term solution is to hard code `params.email` in the aws test config file.

In the long term, hopefully we can set up Slack notifications from Nextflow Tower in the template. See https://github.com/seqeralabs/nf-tower/issues/222

**Comment by ewels** [comment]
Leaving this issue open as I think that it would be good to add a commented out line in the aws config in the template with a `# TODO nf-core:` comment that is picked up by the linting.

**Comment by drpatelh** [comment]
Having it in the config is a good idea but see https://github.com/drpatelh/nf-core-viralrecon/commit/52098845fee86a6d965310ff846361d5699b4e35#commitcomment-39469913 for a possible reason not to do it this way.

**Comment by ewels** [comment]
Good point 👍 

Could still add a `#TODO nf-core:` comment in the GItHub actions workflow file then I guess, explaining what to do.

**Comment by drpatelh** [comment]
Yep 👍 I like the idea of adding the `# TODO nf-core:` directly to `awstest.yml`.

**Comment by drpatelh** [comment]
So as it stands, after a simple test it seems that the instances arent configured correctly or dont have an email client set-up to send an email when the pipeline completes via the `--email` parameter. Will need to look into this :)

**Comment by ewels** [comment]
Sigh, this might be more trouble than it's worth. I guess we need to install either `sendmail` or `mail` in the container, but we may also need to faff around opening up ports in the AMI (if it's like docker?) and the AWS instance network subnet blah blah blah.

This will be much easier if we can rely on Tower to send the email / post to Slack or whatever.

**Comment by ggabernet** [comment]
I found a workaround on how to send email notifications from AWS batch itself in case they fail, etc. Still not the best solution, though, as email addresses need to be independently entered and cannot distinguish between pipelines.

**Comment by ggabernet** [comment]
It looks like you can embed a dashboard from AWS cloudwatch into any website. Could that be something that we could use to monitor the AWS jobs status? 

https://aws.amazon.com/blogs/devops/building-an-amazon-cloudwatch-dashboard-outside-of-the-aws-management-console/
